tests/test_transformers.py

```diff
@@ -80,12 +80,6 @@ def test_resuming(self):
                     num_epochs=1,
                 )
             )
-            # make sure the number of updates is being tracked correctly
```

Comment:
why are these being erased?

Reply:
Sorry fixed. Think that was a master merge gone wrong.